src/main.js

```diff
@@ -77,41 +73,44 @@ function makeUserBook() {
 function saveBook() {
   if (!savedCovers.includes(currentCover)) {
     savedCovers.push(currentCover);
-    savedCoverSection.insertAdjacentHTML('afterbegin', `
-      <div class="mini-cover" id="${currentCover.id}" ondblclick="deleteSavedBook(${currentCover.id})">
-      <img class="mini-cover" src="${currentCover.cover}">
-      <h2 class="cover-title">${currentCover.title}</h2>
-      <h3 class="tagline">A tale of ${currentCover.tagline1} and ${currentCover.tagline2}<h3>
-      <img class="price-tag" src="./assets/price.png">
-      <img class="overlay" src="./assets/overlay.png">
-      </div>
-    `);
+    createHTML();
```

Comment:
Much cleaner! You might consider rewriting this slightly to be something like `insertMiniCover('afterbegin', createHTML())`. And then `createHTML` would return your `htmlBlock` variable.

Not a big change, but you'll probably find yourself using this pattern a lot, and a logical part to break out is whatever constructs this html string from an object (in this case, the object is your currentCover).